README.md

```diff
@@ -185,6 +185,24 @@ serveHTTP(addonInterface, { port: 7000 });
 
 Run the add-on with `npm start` and add `http://127.0.0.1:7000/manifest.json` as the Repository URL in Stremio.
 
+This add-on also supports serverless deployment, just create a new file called serverless.js, that contains:
+
+```javascript
+const finalhandler = require("finalhandler");
```

Comment:
is there a way we can get rid of finalhandler? it should just be a lambda thta returns 404